src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -3176,6 +4107,99 @@ namespace WaterToAirHeatPumpSimple {
         }
     }
 
+    void CheckSimpleWAHPRatedCurvesOutputs(EnergyPlusData &state, std::string const &CoilName)
+    {
+        int WhichCoil;
+        constexpr Real64 Tref(283.15); // Refrence Temperature for performance curves,10C [K]
+        static constexpr std::string_view RoutineName("CheckSimpleWAHPRatedCurvesOutputs");
+
+        WhichCoil = UtilityRoutines::FindItemInList(CoilName, state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP);
+
+        if (WhichCoil != 0) {
+            if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(WhichCoil).WatertoAirHPType == "COOLING") {
+                int TotalCoolCapCurveIndex = state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(WhichCoil).TotalCoolCapCurveIndex;
+                int CoolPowCurveIndex = state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(WhichCoil).CoolPowCurveIndex;
+                int SensCoolCapCurveIndex = state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(WhichCoil).SensCoolCapCurveIndex;
+                if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(WhichCoil).RatedEntAirWetbulbTemp != DataSizing::AutoSize) {
+                    Real64 RatedratioTWB = (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(WhichCoil).RatedEntAirWetbulbTemp +
+                                            state.dataWaterToAirHeatPumpSimple->CelsiustoKelvin) /
```

Comment:
Ummmm ... `CelsiusToKelvin` is a state variable? It's not a `constexpr` somewhere?

Reply:
What's great is that if you fixup the `clear_state` function in WAHPSimple.hh to be `*this = WaterToAirHeatPumpSimpleData();`, you'd get a compiler error because of the const member.  It's a great way I've weeded out members that don't need to be in state.  Because **yes it should just be a constexpr not in state**.

Reply:
Who knows, maybe this is to support alternate universes with different physical constants? Multiverse FTW!